labels/de-DE.json

```diff
@@ -383,7 +384,8 @@
   "mobileShipmentTax": "Steuer",
   "mobileShipmentTitle": "Expédition",
   "mobileShipmentType": "type d'expédition:",
-  "mobileShipping": "livraison:",
+  "mobileShipping": "livraison:",  
```

Comment:
get rid of the changes which you have not made. may be those are showing because of something is added like may be spacing or something. ex-**mobileShipping**

Reply:
resolved